devicetypes/smartthings/zigbee-ceiling-fan-light.src/led-fan-lightings.groovy

```diff
@@ -67,8 +67,8 @@ def parse(String description) {
 	if (event) {
 		sendEvent(event)
 	} else if (description?.startsWith('read attr -')) {
-		if (zigbeeMap.cluster == FAN_CLUSTER_VALUE &&
```

Comment:
Don't bring back magic numbers. As I've already wrote you, you can use `zigbeeMap.clusterInt` to correctly compare number values.
This is the part of `parseDescriptionAsMap(description)` method which proves it. 
```
if (outMap.containsKey("cluster")) {
	outMap["clusterInt"] = convertHexToInt(outMap["cluster"])
}
if (outMap.containsKey("clusterId")) {
	outMap["clusterInt"] = convertHexToInt(outMap["clusterId"])
}
if (outMap.containsKey("attrId")) {
	outMap["attrInt"] = convertHexToInt(outMap["attrId"])
}
if (outMap.containsKey("command")) {
	outMap["commandInt"] = convertHexToInt(outMap["command"])
}
```

Reply:
Are those integers hexadecimal too?

Reply:
Number base actually doesn't matter, you can use both, for example `0x0202` and `514`, but obviously it's better to use first (hex) option in code, as it relates to Zigbee documentation.

Reply:
To extend my response, `0x0202 == 514` would be `true` in Groovy.

Reply:
Oh, thank you for explain. I understand.